test/e2e/conformance_test.go

```diff
@@ -99,6 +98,11 @@ func TestClusterConformance(t *testing.T) { //nolint:gocyclo
 		// to satisfy scope linter
 		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
+			_, runThisTest := os.LookupEnv("TEST_SET")
+			if !runThisTest {
+				t.Skip("set TEST_SET to run this test")
+			}
```

Comment:
Can we namespace this environment variable, i.e. call it something like `KUBEONE_TEST_SET`?